synthtool/gcp/templates/node_library/.github/CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+# The yoshi-nodejs team is the default owner for nodejs repositories.
+{% if 'codeowner_team' in metadata['repo'] %}
+*  @googleapis/yoshi-nodejs {{ metadata['repo']['codeowner_team'] }}
```

Comment:
I love this idea. I'm trying to remember if we have any CODEOWNER files that have multiple owners?

Reply:
The only instance I saw of this was firestore, and we're going to have to figure out that one.  Worst case, we can opt out of updates like we do for things like kokoro in some places.